webknossos-datastore/app/com/scalableminds/webknossos/datastore/services/UploadService.scala

```diff
@@ -224,12 +228,50 @@ class UploadService @Inject()(dataSourceRepository: DataSourceRepository,
       Fox.successful(())
     else {
       for {
-        _ <- addLayerAndResolutionDirIfMissing(unpackToDir).toFox
-        _ <- addSymlinksToOtherDatasetLayers(unpackToDir, layersToLink.getOrElse(List.empty))
-        _ <- addLinkedLayersToDataSourceProperties(unpackToDir, dataSourceId.team, layersToLink.getOrElse(List.empty))
+        isZarr <- looksLikeZarrArray(unpackToDir).toFox
```

Comment:
Do you think we could detect existing datasource-properties.jsons and in this case skip the explorer (or possibly, just make the paths relative)?

I tried uploading the sharded zarr3 dataset from [here](https://www.notion.so/scalableminds/Test-Datasets-c0563be9c4a4499dae4e16d9b2497cfb?pvs=4#fea01b638d2c4c689565adb72a79630a), which webknossos can just serve as is. However, the upload was rejected. Since this (or something close) is to become our standard format, which will also be created by the wkcuber, uploading this should also work.